xeofs/models/opa.py

```diff
@@ -135,7 +140,7 @@ def _fit_algorithm(self, data: DataArray) -> Self:
             solver=self._params["solver"],
             compute=self._params["compute"],
             random_state=self._params["random_state"],
-            solver_kwargs=self._solver_kwargs,
+            solver_kwargs=self._params["solver_kwargs"],
```

Comment:
OPA can also enjoy your new feature :) 

```suggestion
            check_nans=False,
            solver_kwargs=self._params["solver_kwargs"],
```